test/sqrt.jl

```diff
@@ -0,0 +1,44 @@
+using PDMats
+using Test
+using LinearAlgebra: LinearAlgebra
+
+_randPDMat(T, n) = (X = randn(T, n, n); PDMat(X * X' + LinearAlgebra.I))
+_randPDiagMat(T, n) = PDiagMat(rand(T, n))
+_randScalMat(T, n) = ScalMat(n, rand(T))
+_randPDSparseMat(T, n) = (X = T.(sprand(n, 1, 0.5)); PDSparseMat(X * X' + LinearAlgebra.I))
```

Comment:
Most of these are already define in https://github.com/JuliaStats/PDMats.jl/blob/84df00b41da9573c3e0b7e79bdcde2dda24bb2d5/test/kron.jl#L5-L7. Can you avoid the method redefinition and move these methods to tests/testutils.jl?

Reply:
Good spot. Have moved to `testutils.jl`